hi-ml-histopathology/src/histopathology/utils/report_utils.py

```diff
@@ -15,8 +15,51 @@
                                 get_tags_from_hyperdrive_run)
 
 
+def run_has_val_and_test_outputs(run: Run) -> bool:
+    """Checks whether the given run has both validation and test outputs files.
+
+    :param parent_run: The run whose outputs to check.
+    :raises ValueError: If the run does not have the expected output file(s).
+    :return: `True` if the run has validation and test outputs, `False` if it is a legacy run with
+        only test outputs.
+    """
+    outputs_filename = "test_output.csv"
+    val_outputs_filename = "val/" + outputs_filename
+    test_outputs_filename = "test/" + outputs_filename
+
+    outputs_folder = "outputs/"
+    all_available_files: List[str] = run.get_file_names()
```

Comment:
It would be nice to use `pathlib.Path` in this function, instead of strings. Then, once you've defined the output, test and val folder, you can just `return test_outputs_path.is_file() and val_outputs_path.is_file()`.

Reply:
Normally I would indeed use `Path`s, but these are files in an AML run; they don't exist on the local machine.

Reply:
Of course 🤦‍♂️.